includes/DataDump.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 use MediaWiki\MediaWikiServices;
+use WikiMap;
```

Comment:
`use` statement is unnecessary since this is not a namespaced class.

Reply:
Done